test/fixtures/CHANGELOG.md

```diff
@@ -13,5 +45,3 @@
 ## 0.1.0 - 2017-07-21
 
 * Very old version
-
-##
```

Comment:
So this issue is fixed ?
It was somehow required if the changelog has less than 3 versions.

Reply:
Yes, I think so. Anyway, the news tests are passing. Please check it on your side and tell us.